File: Daily Challenge/Python/Flowchart Logic 063 23.12.2025.py
num = int(input())
val = 0
count = 0

while num > 0:
    if num % 10 != 0:
        val = val * 10 + (num % 10)
    else:
        count += 1
    num //= 10

while count > 0:
    val *= 10
    count -= 1

print(val)

File: Daily Challenge/C/Flowchart Logic 063 23.12.2025.c
#include <stdio.h>

int main() {
    int num, val = 0, count = 0;
    scanf("%d", &num);

    while (num > 0) {
        if (num % 10 != 0) {
            val = val * 10 + (num % 10);
        } else {
            count += 1;
        }
        num /= 10;
    }

    while (count > 0) {
        val = val * 10;
        count -= 1;
    }

    printf("%d
", val);
    return 0;
}